packages/eas-cli/src/submissions/ios/AppProduce.ts

```diff
@@ -64,11 +52,7 @@ Learn more here: https://expo.fyi/bundle-identifier`
     language: sanitizeLanguage(language),
   };
 
-  if (USE_APPLE_UTILS) {
-    return await runProduceExperimentalAsync(options);
-  }
-
-  return await runProduceAsync(options);
+  return await runProduceExperimentalAsync(options);
```

Comment:
rename method

Reply:
What should it be called now? `createAppStoreConnectAppAsync`? (I guess `Produce` is a fastlaneism.)